docs/medical-api/handling-data/cohorts.mdx

```diff
@@ -0,0 +1,177 @@
+---
+
+# Overview
+
+Cohorts allow you to configure shared patient settings. 
+For example, you could create one cohort to monitor a group of 
+chronically-ill individuals with all features enabled (e.g., ADTs, pharmacies, laboratory, HIE), and another cohort 
+that monitors a less critical group of individuals with just HIE data enabled.
+
+# Getting started
+
+Every cohort has a unique name and a collection of settings which apply to all patients within that cohort.
+By default, you have 3 cohorts with the following names + behaviors:
+
+- 🟢 Low Risk: Monthly medical record refresh & no additional monitoring 
+- 🟡 Medium Risk: Monthly medical record refresh & prescription medication history  
+- 🔴 High Risk: Weekly medical record refresh & monitoring enabled for all available sources
+
+<Note>
+  All of this can be personalized based on your needs,
+  either during onboarding or self-serve, via our API (See
+  our API reference for documentation on how to [update an
+  existing
+  cohort](/medical-api/api-reference/cohort/update-cohort)
+  or [add existing patients to a
+  cohort](/medical-api/api-reference/cohort/add-patients-to-cohort)).
+</Note>
+
+# The Cohort Model
+
+In Metriport, a cohort is simply a label that can be given to any number of patients. You can 
+use that label to apply common settings across a group of patients,
+or retrieve the members of that cohort via API.
+
+```json
+{
+  "name": "High Risk",
+  "description": "Members of this cohort are opted-in to all available data sources.",
+  "color": "red",
+  "settings": {
+    "monitoring": {
+      "adt": true,
+      "hie": {
+        "enabled": true,
+        "frequency": "weekly"
+      },
+      "pharmacy": {
+        "notifications": true,
+        "schedule": {
+          "enabled": false,
+          "frequency": "monthly"
+        }
+      },
+      "laboratory": {
+        "notifications": true,
+        "schedule": {
+          "enabled": false,
+          "frequency": "monthly"
+        }
+      }
+    }
+  }
+}
+```
+See [Here](/medical-api/handling-data/cohorts#available-settings) for a detailed explanation of the available settings.
+
+# Monitoring patients with Cohorts
+
+Metriport allows you to monitor patients and receive notifications about them over time.
+
+To do this, create a cohort and specify the settings directly on the cohort. Then add patients that need to be monitored to the cohort. 
+The settings of the cohort are now applied to the patient (what monitoring is set, schedule, etc.).
+
+```typescript
+// 1. Create a cohort
+const cohort = await metriportApi.createCohort({
+  name: "My Custom Cohort",
+  description: "This is my first cohort",
+  color: "purple",
+  settings: {
+    monitoring: {
+      adt: true
+    }
+  }
+ });
+
+// 2. Add patients to the cohort
+const patientsInCohort = await metriportApi.addPatientsToCohort({
+  cohortId: cohort.id,
+  patientIds: [
+    "patient-uuid-1",
+    "patient-uuid-2",
+    "patient-uuid-3",
```

Comment:
I thought the decision was to use UUIDs based on 1111, 2222, 3333 when we wanted to have many diff UUIDs on the same example? Like this:

```
"11111111-1111-1111-1111-111111111111",
"22222222-2222-2222-2222-222222222222"
```